src/TableComponent/Table.tsx

```diff
@@ -276,15 +284,13 @@ function GraaspTable<T>({
         {/* TODO: implement pagination */}
         {pagination && (
           <TablePagination
-            count={rowData.length}
+            count={totalCount ? totalCount : rowData.length}
             labelDisplayedRows={labelDisplayedRows}
-            page={0}
+            page={isClientPagination ? 0 : page}
+            onPageChange={onPageChange ? onPageChange : () => {}}
             rowsPerPageOptions={[]}
-            onPageChange={() => {
-              // todo
-            }}
             component='div'
-            rowsPerPage={-1}
+            rowsPerPage={isClientPagination ? -1 : 10}
```

Comment:
`10` comes from your default limit right? I think it should be a prop as well.
Also why not use https://github.com/graasp/graasp-sdk to have the default value (to be reused in the backend as well)?